### ISSUE TITLE
week1 - ex1

### ISSUE THREAD

**Issue by littlelittlemoon** [littlelittlemoon/Andrew-Ng-C-MachineLearning#1]
## Outline
- Gradient descent
  - Update Equations
  - Computing the cost J(θ)
- Linear regression with one variable
- Linear regression with multiple variables
  - Feature Normalization
  - Selecting learning rates
  - Normal Equations
- Octave/MATLAB
